### DISCUSSION TITLE
GHelper v0.157 Microsoft Defender says it has a virus in

### DISCUSSION THREAD

**Discussion by JV-bug** [seerge/g-helper#2178] [General]
GHelper v0.157 Microsoft Defender says it has a virus in, why?

**Comment by seerge** [comment]
https://github.com/seerge/g-helper/wiki/FAQ#windows-defender-marks-app-as-malware--virus
> Windows Defender marks app as malware / virus
> False positives from Windows Defender (or any other similar system that uses machine learning for detection) is possible as the application is not digitally signed with a certificate. You can always download a version below or compile the app by yourself. All application sources are open and can be monitored from A to Z :)
